ansible/roles/gitlab/tasks/configure_gitlab_ce.yml

```diff
@@ -271,6 +271,9 @@
   cron:
     name: 'Create GitLab backup'
     special_time: '{{ gitlab_backup_frequency }}'
+    disabled: '{{ False
+                  if (gitlab_backup_enabled|d(True))
```

Comment:
The `|d(True)` here is meaningless because the `gitlab_backup_enabled` variable is a default variable which will always be defined in the role context. I would change this to `|bool` instead to check for the boolean value.